README.md

```diff
@@ -343,6 +344,15 @@ Found '/path/to/project/.nvmrc' with version <5.9>
 Now using node v5.9.1 (npm v3.7.3)
 ```
 
+### .node-version
+
+For a little compatability with other node version managers, nvm will also sniff for `.node-version` files. They're the same as `.nmvrc`, they just share a common name.
+
+$ echo "5.9" > .node-version
+
+They'll be loaded after `.nvmrc`, and can contain the same values as `.nvmrc`.
```

Comment:
"same values" can't be correct; nvmrc supports both custom and built-in aliases, and node-version doesn't.

Also, nvmrc can have "iojs-" prefixes; but node-version can't.

Reply:
Please offer replacement text for this. I'm not a nvmrc user.

Reply:
How about:

> Unlike `.nvmrc`, `.node-version` can not contain a “versionish” (an alias, like “node”, “iojs”, or a custom alias you’ve defined). `.node-version` can only have versions in the format of `v1`, `v1.2`, or `v1.2.3` (the “v” is optional).